tests/unit/OrgPerson.spec.ts

```diff
@@ -16,238 +13,289 @@ const vuetify = new Vuetify({})
+
     wrapper.destroy()
   })
 
-  it('Displays form data for person', async () => {
-    const wrapper: Wrapper<OrgPerson> = createComponent(validPersonData, 0, 0, null)
-    expect((<HTMLInputElement> wrapper.find(firstNameSelector).element).value)
-      .toEqual(validPersonData['officer']['firstName'])
-    expect((<HTMLInputElement> wrapper.find(middleNameSelector).element).value)
-      .toEqual(validPersonData['officer']['middleName'])
-    expect((<HTMLInputElement> wrapper.find(lastNameSelector).element).value)
-      .toEqual(validPersonData['officer']['lastName'])
+  it('Displays form data for org', async () => {
+    const wrapper = createComponent(validOrgData, NaN, 0, null)
     await Vue.nextTick()
+
+    expect((wrapper.find(orgNameSelector).element as HTMLInputElement).value)
+      .toEqual(validOrgData['officer']['orgName'])
+
     expect(wrapper.find(doneButtonSelector).attributes('disabled')).toBeUndefined()
-    expect(wrapper.find(removeButtonSelector).attributes('disabled')).toBeUndefined()
+    expect(wrapper.find(removeButtonSelector).attributes('disabled')).toBeDefined()
     expect(wrapper.find(cancelButtonSelector).attributes('disabled')).toBeUndefined()
+
     wrapper.destroy()
   })
 
   it('Remove button is enabled in edit mode', async () => {
-    const wrapper: Wrapper<OrgPerson> = createComponent(validOrgData, 0, 0, null)
+    const wrapper = createComponent(validOrgData, 0, null, null)
+
     expect(wrapper.find(removeButtonSelector).attributes('disabled')).toBeUndefined()
+
     wrapper.destroy()
   })
 
-  xit('Remove button is disabled in create mode', async () => {
-    const wrapper: Wrapper<OrgPerson> = createComponent(validOrgData, -1, 0, null)
+  it('Remove button is disabled in create mode', async () => {
+    const wrapper = createComponent(validOrgData, NaN, 0, null)
+
     expect(wrapper.find(removeButtonSelector).attributes('disabled')).toBeDefined()
+
     wrapper.destroy()
   })
 
-  xit('Clicking remove button emits event', async () => {
-    const wrapper: Wrapper<OrgPerson> = createComponent(validOrgData, 0, 0, null)
+  it('Clicking Remove button emits "remove" event', async () => {
+    const wrapper = createComponent(validOrgData, 0, null, null)
+
     wrapper.find(removeButtonSelector).trigger('click')
     await Vue.nextTick()
-    expect(getLastEvent(wrapper, removePersonEvent)).toBe(0)
+
+    expect(getLastEvent(wrapper, removeEvent)).toBe(0)
+
     wrapper.destroy()
   })
 
-  xit('Clicking Done button emits event for add edit person/org', async () => {
-    const wrapper: Wrapper<OrgPerson> = createComponent(validOrgData, -1, 0, null)
-    expect((<HTMLInputElement> wrapper.find(orgNameSelector).element).value)
-      .toEqual(validOrgData['officer']['orgName'])
+  it('Clicking Done button when org not changed emits "reset" event', async () => {
+    const wrapper = createComponent(validOrgData, 0, null, null)
     await Vue.nextTick()
```

Comment:
This is worth pointing out: after mounting a component, most times you have to wait for things to settle (ie, for Vuetify to finish updating the DOM).